package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "testexec-navigator",
+  "version": "0.0.0",
```

Comment:
what does this imply for the first released version (if anything)? I suppose we'd want to start with version 0.1.0 or something?